packages/styles/src/modules/typography.css

```
@@ -0,0 +1,90 @@
+/**
+ * Core typographic styles
+ */
+
+/**
+ * Utility class for setting a very small body copy font style
+ * May be combined with weight classes defined above
+ */
+.syn-body-x-small {
+  font: var(--syn-body-x-small-regular);
+}
+
+/**
+ * Utility class for setting a small body copy font style
+ * May be combined with weight classes defined above
+ */
+.syn-body-small {
+  font: var(--syn-body-small-regular);
+}
+
+/**
+ * Utility class for setting a medium body copy font style
+ * This is the default style for elements for synergy applications
+ * May be combined with weight classes defined above
+ */
+.syn-body-medium {
+  font: var(--syn-body-medium-regular);
+}
+
+/**
+ * Utility class for setting a large body copy font style
+ * May be combined with weight classes defined above
```

The weight classes are defined below:

```suggestion
 * May be combined with weight classes defined below
```